.gitignore

```diff
@@ -4,4 +4,6 @@ documentation/
 *~
 *swp
 *.user*
+.vs
+*.bak
```

Comment:
Fair enough for VS, but in general users should configure this in their global local git config. We can't add settings for every IDE in existence to a project